kodi_addon_checker/check_addon.py

```diff
@@ -55,6 +56,9 @@ def start(addon_path, branch_name, all_repo_addons, pr, config=None):
             check_entrypoint.check_complex_addon_entrypoint(
                 addon_report, addon_path, parsed_xml, max_entrypoint_count)
 
+            if branch_name == "leia":
```

Comment:
no `branch_name` checks

Reply:
I thought @Razzeee said we should run this only for branches above `leia`

Reply:
That's not what your code does.
IMO it should be run on all branches (or at least kyrpton and higher), but type of report (info and error) should be different.